ui/options_folksonomy.ui

```diff
@@ -1,68 +1,79 @@
-<ui version="4.0" >
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
  <class>FolksonomyOptionsPage</class>
- <widget class="QWidget" name="FolksonomyOptionsPage" >
-  <property name="geometry" >
+ <widget class="QWidget" name="FolksonomyOptionsPage">
+  <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>390</width>
+    <width>590</width>
     <height>304</height>
    </rect>
   </property>
-  <layout class="QVBoxLayout" name="verticalLayout_2" >
+  <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
-    <widget class="QGroupBox" name="rename_files_3" >
-     <property name="title" >
+    <widget class="QGroupBox" name="rename_files_3">
+     <property name="title">
       <string>Folksonomy Tags</string>
      </property>
-     <layout class="QVBoxLayout" name="verticalLayout" >
+     <layout class="QVBoxLayout" name="verticalLayout">
       <item>
-       <widget class="QLabel" name="ignore_tags_2" >
-        <property name="text" >
+       <widget class="QLabel" name="ignore_tags_2">
+        <property name="text">
          <string>Ignore tags:</string>
         </property>
        </widget>
       </item>
       <item>
-       <widget class="QLineEdit" name="ignore_tags" />
+       <widget class="QLineEdit" name="ignore_tags"/>
       </item>
       <item>
-       <widget class="QCheckBox" name="only_my_tags" >
-        <property name="text" >
+       <widget class="QCheckBox" name="only_my_tags">
+        <property name="text">
          <string>Only use my tags</string>
         </property>
        </widget>
       </item>
       <item>
-       <layout class="QHBoxLayout" >
-        <property name="spacing" >
+       <widget class="QCheckBox" name="artists_tags">
+        <property name="enabled">
+         <bool>true</bool>
+        </property>
+        <property name="text">
+         <string>Fallback on album's artits tags if no tags are found for the release or release group</string>
```

Comment:
I prefer to keep it as an option, disabled by default, so the current behavior is preserved.
Also i think it could be problematic for some artists which aren't in a particular genre, ie. if one is tagged "rap,blues,electronic", and released one album of each genre, this feature may tag a blues album as genre rap, etc.